src/plugins/linked.js

```diff
@@ -110,14 +118,15 @@ export default class LinkedPlugins implements IPluginManager {
     if (pjson.scripts && pjson.scripts.prepare) {
       if (!this.config.debug) this.out.action.start(`Running prepare script for ${p}`)
       await this.yarn.exec(['run', 'prepare'], {cwd: p})
-      fs.utimesSync(main, new Date(), new Date())
       this.out.action.stop()
     }
+    touch(main)
 
     return true
   }
 
   _save () {
+    this._data.updated_at = new Date()
```

Comment:
@dickeyxxx could you move this up to https://github.com/heroku/cli-engine/blob/700a51165ecf07e5ec50740e5dbf4d25123fecc9/src/plugins/linked.js#L103 ?  When someone does an `add` or `remove` of a linked plugin, it will bump the updated at which could cause it to miss a linked plugin change?